### ISSUE TITLE
Cannot tap on UITabBarButton

### ISSUE THREAD

**Issue by ghost** [kif-framework/KIF#457]
```
 I am having trouble getting KIF to tap on a UITabBarItem.  The title of the button is dynamic, so I have to use the accessibility label.  However, the call to "tapViewWithAccessibilityLabel" works only if I used the UITabBarItem title.
 For testing I tried this:
       [tabBarItem setTitle:@"Title"];
       [tabBarItem setAccessibilityLabel:@"AccessibilityLabel"];
```

If I call:
         [tester tapViewWithAccessibilityLabel:@"AccessibilityLabel"];
I get the error:
         Failed to find accessibility element with the label "AccessibilityLabel"
However, if I call:
         [tester tapViewWithAccessibilityLabel:@"Title"];
then the method succeeds.

I found an old issue with this on google groups (https://groups.google.com/forum/#!msg/kif-framework/dxpwLPvKKfo/3G8_8cYiH_oJ).

I'm wondering if there is an update.
